e2e/vite/src/vite-ts-solution.test.ts

```diff
@@ -103,10 +103,5 @@ ${content}`
     expect(runCLI(`build ${reactApp}`)).toContain(
       `Successfully ran target build for project @proj/${reactApp} and 5 tasks it depends on`
     );
-
-    // check typecheck
-    expect(runCLI(`typecheck ${reactApp}`)).toContain(
-      `Successfully ran target typecheck for project @proj/${reactApp} and 6 tasks it depends on`
-    );
```

Comment:
Why is this removed? The app uses the `vite` bundler, so it should have a typecheck task. Or is it failing due to a different issue?